### ISSUE TITLE
Hyperparameter optimization

### ISSUE THREAD

**Issue by jeffjennings** [MPoL-dev/MPoL#135]
**Is your feature request related to a problem or opportunity? Please describe.**
Try raytune or similar to search parameter space in at least somewhat optimal way to determine regularizer strengths and ideally also learning rate (probably a simple way to do that).

**Comment by jeffjennings** [comment]
Closing as out of scope with v0.3 redesign
